scripts/smoke-test-litellm.sh

```
@@ -0,0 +1,74 @@
+STATUS=$(curl -s -o /dev/null -w "%{http_code}" \
+  -X POST "$BASE_URL/v1/chat/completions" \
+  -H "Content-Type: application/json" \
+  -d '{"model":"mistral-small","messages":[{"role":"user","content":"ping"}]}')
+[[ "$STATUS" == "401" ]] && pass "Unauthenticated POST /v1/chat/completions → $STATUS" || fail "Expected 401, got $STATUS (auth not enforced!)"
+
+# 3. Models list
+echo "[3/5] Models list..."
+RESPONSE=$(curl -s "$BASE_URL/models" -H "Authorization: Bearer $MASTER_KEY")
+echo "$RESPONSE" | grep -q '"mistral-small"' && pass "GET /models contains mistral-small" || fail "mistral-small missing from /models response"
+echo "$RESPONSE" | grep -q '"mistral-large"' && pass "GET /models contains mistral-large" || fail "mistral-large missing from /models response"
+echo "$RESPONSE" | grep -q '"mistral-codestral"' && pass "GET /models contains mistral-codestral" || fail "mistral-codestral missing from /models response"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)